backend/config/initializers/cors.rb

```diff
@@ -1,17 +1,18 @@
 # frozen_string_literal: true
+
 # Be sure to restart your server when you modify this file.
 
 # Avoid CORS issues when API is called from the frontend app.
 # Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
 
 # Read more: https://github.com/cyu/rack-cors
 
-# Rails.application.config.middleware.insert_before 0, Rack::Cors do
-#   allow do
-#     origins "example.com"
-#
-#     resource "*",
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before(0, Rack::Cors) do
+  allow do
+    origins "http://localhost:3000"
```

Comment:
FYI that `origins` can receive an array of strings. 

I recommend moving the accepted origins in the env variable as an array of strings. So that it'll be easier to add multiple origins if needed